File: app/src/main/java/com/grin/appforuniver/data/model/dto/ConsultationRequestDto.java
package com.grin.appforuniver.data.model.dto;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

public class ConsultationRequestDto {

    @SerializedName("idCreatedUser")
    private int mIdCreatedUser;

    @SerializedName("idRoom")
    private int mIdRoom;

    @SerializedName("dateOfPassage")
    private String dateOfPassage;

    @SerializedName("description")
    private String description;

    public ConsultationRequestDto(int idCreatedUser, int idRoom, String dateOfPassage, String description) {
        this.mIdCreatedUser = idCreatedUser;
        this.mIdRoom = idRoom;
        this.dateOfPassage = dateOfPassage;
        this.description = description;
    }
}

File: app/src/main/java/com/grin/appforuniver/data/models/dto/ConsultationRequestDto.kt
package com.grin.appforuniver.data.models.dto

import com.google.gson.annotations.SerializedName

data class ConsultationRequestDto(
        @SerializedName("idCreatedUser") var idCreatedUser: Int? = null,
        @SerializedName("idRoom") var idRoom: Int? = null,
        @SerializedName("dateOfPassage") var dateOfPassage: String? = null,
        @SerializedName("description") var description: String? = null
) {
}